File: __tests__/__fixtures__/output/node/axios/multipart-form-data-no-params.js
const axios = require("axios").default;

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  headers: {'Content-Type': 'multipart/form-data'}
};

axios.request(options).then(function (response) {
  console.log(response.data);
}).catch(function (error) {
  console.error(error);
});

File: src/targets/swift/urlsession/fixtures/multipart-form-data-no-params.swift
import Foundation
#if canImport(FoundationNetworking)
  import FoundationNetworking
#endif

let headers = ["Content-Type": "multipart/form-data"]

var request = URLRequest(url: URL(string: "https://httpbin.org/anything")!)
request.httpMethod = "POST"
request.allHTTPHeaderFields = headers

let (data, response) = try await URLSession.shared.data(for: request)
print(String(decoding: data, as: UTF8.self))
